src/wizard.scss

```diff
@@ -130,6 +116,20 @@ $fd-wizard-stacked-upcoming-step-offset: 2.375rem !default;
       }
     }
 
+    &--completed {
+      .#{$block}__step-indicator {
+        background: var(--sapObjectHeader_Background);
+        color: var(--fdWizard_Completed_Step_Color);
+        border: var(--fdWizard_Completed_Step_Border);
+
+        .#{$block}__icon {
+          @include fd-icon-selector() {
+            color: var(--fdWizard_Completed_Step_Icon_Color);
+          }
+        }
+      }
+    }
```

Comment:
@N1XUS what is the change here? You moved the `--completed` modifier to be after `--upcoming`?

Reply:
Yes, my intention is to keep the `upcoming` class with `completed`. The logic here is that even if the step is completed, but we moved backward, it will be upcoming to refill/skip. By moving the code, I've changed the priority of the styling, which should do the trick with the ngx PR

Reply:
I think this is an unsafe way to address the problem that is prone to future problems, we should either create an additional class or have more specificity for "completed but also upcoming"

Reply:
Agree with Mike. We experienced multiple problems in the past due to the order of the classes in the file or the order of file import. This can be easily solved with specificity.

Reply:
Done. Please re-check.